org.skytemple.SkyTemple.yml

```diff
@@ -0,0 +1,144 @@
+app-id: org.skytemple.SkyTemple
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: /app/bin/run.sh
+finish-args:
+  - "--share=network"
+  - "--socket=fallback-x11"
+  - "--socket=wayland"
+  - "--filesystem=host"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-nightly
+cleanup:  # todo
+  - /bin/pcap-config
+  - /include
+  - /lib/libpcap.a
+  - /lib/pkgconfig
+  - /share/man
+modules:
+  - name: armips
+    buildsystem: simple
+    build-commands:
+      - mkdir -p build
+      - patch Core/ExpressionFunctions.cpp < armips.patch
+      - cd build && cmake -DCMAKE_BUILD_TYPE=Release ..
+      - cd build && make armips-bin
+      - install -Dm755 build/armips /app/bin/armips
+    sources:
+      - type: archive
+        url: https://github.com/Kingcom/armips/archive/v0.11.0.tar.gz
+        sha256: c94e29dfda3bdf853590d825562b9c73a3d8e8e995555e021c6b2a6451572681
+      - type: file
+        path: patches/armips.patch
+  - name: skytemple
+    buildsystem: simple
+    build-commands:
+      - pip3 install --prefix=/app --no-deps  .
+      # Icons
+      - mkdir -p /app/share/icons/hicolor/16x16/apps
+      - install -Dm644 skytemple/data/icons/hicolor/16x16/apps/skytemple.png /app/share/icons/hicolor/16x16/apps/org.skytemple.SkyTemple.png
+      - mkdir -p /app/share/icons/hicolor/32x32/apps
+      - install -Dm644 skytemple/data/icons/hicolor/32x32/apps/skytemple.png /app/share/icons/hicolor/32x32/apps/org.skytemple.SkyTemple.png
+      - mkdir -p /app/share/icons/hicolor/64x64/apps
+      - install -Dm644 skytemple/data/icons/hicolor/64x64/apps/skytemple.png /app/share/icons/hicolor/64x64/apps/org.skytemple.SkyTemple.png
+      - mkdir -p /app/share/icons/hicolor/128x128/apps
+      - install -Dm644 skytemple/data/icons/hicolor/128x128/apps/skytemple.png /app/share/icons/hicolor/128x128/apps/org.skytemple.SkyTemple.png
+      - mkdir -p /app/share/icons/hicolor/256x256/apps
+      - install -Dm644 skytemple/data/icons/hicolor/256x256/apps/skytemple.png /app/share/icons/hicolor/256x256/apps/org.skytemple.SkyTemple.png
+      - mkdir -p /app/share/icons/hicolor/512x512/apps
+      - install -Dm644 skytemple/data/icons/hicolor/512x512/apps/skytemple.png /app/share/icons/hicolor/512x512/apps/org.skytemple.SkyTemple.png
```

Comment:
```suggestion
      - install -Dm644 skytemple/data/icons/hicolor/16x16/apps/skytemple.png /app/share/icons/hicolor/16x16/apps/org.skytemple.SkyTemple.png
      - install -Dm644 skytemple/data/icons/hicolor/32x32/apps/skytemple.png /app/share/icons/hicolor/32x32/apps/org.skytemple.SkyTemple.png
      - install -Dm644 skytemple/data/icons/hicolor/64x64/apps/skytemple.png /app/share/icons/hicolor/64x64/apps/org.skytemple.SkyTemple.png
      - install -Dm644 skytemple/data/icons/hicolor/128x128/apps/skytemple.png /app/share/icons/hicolor/128x128/apps/org.skytemple.SkyTemple.png
      - install -Dm644 skytemple/data/icons/hicolor/256x256/apps/skytemple.png /app/share/icons/hicolor/256x256/apps/org.skytemple.SkyTemple.png
      - install -Dm644 skytemple/data/icons/hicolor/512x512/apps/skytemple.png /app/share/icons/hicolor/512x512/apps/org.skytemple.SkyTemple.png
```

Reply:
Thanks, somehow assumed install wouldn't create the directories for some reason.

Reply:
but you put the `-D` option.